### ISSUE TITLE
Grammar issue on: http://www.diveintopython3.net/regular-expressions.html#summary

### ISSUE THREAD

**Issue by cezaryzukowski** [diveintomark/diveintopython3#50]
On http://www.diveintopython3.net/regular-expressions.html#summary there is:

"* x? matches an optional x character (in other words, it matches an x zero or one times)."

Should it not be "one time" without 's' character (form more suitable grammatically)?

**Comment by alecthegeek** [comment]
Or ""* x? matches an optional x character (in other words, it matches an x zero or once only)."

**Comment by cezaryzukowski** [comment]
Exactly, "once only" may suit or anything else indicating the singular and that it is single. 

As far as I remember the grammar concerning "and" and "or" statements:
- AND implies the plural ("s" attached), 
- OR implies the singular (without "s").

So,
- OR - "zero or one time" or "zero or once only" - is grammatically correct (without "s"),
- AND -  "zero and one times" - is still grammatically correct (the plural form), but sounds odd logically since both "zero and one times" can not occur at the same time.

I know it's a tiny issue; therefore, do not bother about it overly much.
